css/props.css

```diff
@@ -66,3 +69,20 @@ html {
   --ccs-special--fg3: hsl(24deg 53% 27%);
   --ccs-accent--fg2: hsl(330deg 60% 35%);
 }
+
+@supports (font-size: 2cqi) {
```

Comment:
So Safari 16 supports cqi, but it looks like it doesn't like the lh unit so spacer gets ignored. Still testing Safari 15 ... 

![Screenshot 2025-03-07 at 10 57 23 AM](https://github.com/user-attachments/assets/c3b6175b-31bb-42a8-a47f-3505a46e6b0c)
![Screenshot 2025-03-07 at 10 51 25 AM](https://github.com/user-attachments/assets/e8b174aa-d0bb-4ac3-b6dc-dd2793ab5861)

Reply:
good catch. fixed in https://github.com/oddbird/popover-polyfill/pull/236/commits/5ef1a52d3f3b1edf120ad7b6b27d287c711cdd78

Safari 16: 
https://github.com/oddbird/popover-polyfill/pull/236/commits/5ef1a52d3f3b1edf120ad7b6b27d287c711cdd78

![image](https://github.com/user-attachments/assets/de523492-2a2f-4a72-bbc5-acf8d3ad5891)

![image](https://github.com/user-attachments/assets/0c905330-9194-4626-91ed-083b0f9f8e5c)

And 15:
![image](https://github.com/user-attachments/assets/9919f3e4-2433-4d87-aba5-1d80042f4de5)

![image](https://github.com/user-attachments/assets/35ee6d06-7c4b-4505-893d-d4366a489a2b)